### PR TITLE
Wait for JMS restart after config change

### PR DESCRIPTION
**Pull request by AndrewDJBanks**

### CODE REVIEW

**Comment by AndrewDJBanks** [comment]
#build

**Comment by AndrewDJBanks** [comment]
#build

**Comment by AndrewDJBanks** [comment]
#build

**Comment by AndrewDJBanks** [comment]
#build

**Comment by AndrewDJBanks** [comment]
#build

File: dev/com.ibm.ws.messaging.common/src/com/ibm/ws/messaging/lifecycle/SingletonsReady.java [hunk 1]
```diff
@@ -145,7 +144,7 @@ public SingletonsReady(
             debug(this, tc, "Messaging singletons are now available: ", impls);
             debug(this, tc, "Messaging singletons also available as: ", intfs);
         }
-        if (isAnyTracingEnabled() && tc.isEntryEnabled()) exit(this, tc, "<init>");
+        if (isAnyTracingEnabled() && tc.isEntryEnabled()) exit(this, tc, "<init>", "System.identityHashCode="+System.identityHashCode(SingletonsReady.class));
```
Comment (joe-chacko) [inline dev/com.ibm.ws.messaging.common/src/com/ibm/ws/messaging/lifecycle/SingletonsReady.java:147]: Did you mean to leave this trace in?

**Review by joe-chacko — APPROVED** [review]

**Review by joe-chacko — DISMISSED** [review]

**Comment by AndrewDJBanks** [comment]
#build

**Review by joe-chacko — APPROVED** [review]

**Comment by joe-chacko** [comment]
#run-libby-bot

**Pull request merged**